start/server/src/index.js

```diff
@@ -1,7 +1,7 @@
 const { ApolloServer } = require('apollo-server');
 const typeDefs = require('./schema');
 const { createStore } = require('./utils');
-const { resolvers } = require('./resolvers');
+const resolvers = require('./resolvers');
```

Comment:
This `require` was the main issue. If you have a file you're requiring, that require needs to mirror how you've structured the *module.exports* in the required file. When you write, `const { resolvers } = require('./resolvers')`, you're saying, "Reach inside `./resolvers.js`, and pull in the value with the key `resolvers` on `module.exports`". 

For example, in `./resolvers./js`, if the export was done using the `resolvers` key:

```javascript
module.exports.resolvers = {
  Query: {
    launches:
    // more code here
   }
};
```

Then the original require of `const { resolvers } = require('./resolvers');` would have worked.

But resolvers actually exports this way:

```javascript
module.exports = {
  Query: {
    launches:
    // more code here
   }
};
```

Which means the require should be `const resolvers = require('./resolvers')`.